src/AB1805_RK.cpp

```diff
@@ -39,12 +39,12 @@ void AB1805::setup(bool callBegin) {
 
             _log.info("set system clock from RTC %s", Time.format(time, TIME_FORMAT_DEFAULT).c_str());
         }
-    }
-    else {
+        System.on(reset, systemEventStatic);
```

Comment:
Im fine w this but its interesting how if we failed to detect the chip we still wrote to registers right before a reset

Reply:
yeah lol